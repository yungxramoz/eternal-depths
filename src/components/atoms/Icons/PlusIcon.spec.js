import React from 'react'
import TestRenderer from 'react-test-renderer'
import PlusIcon from './PlusIcon'

describe('PlusIcon', () => {
  it('renders correctly', () => {
    const tree = TestRenderer.create(<PlusIcon />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
