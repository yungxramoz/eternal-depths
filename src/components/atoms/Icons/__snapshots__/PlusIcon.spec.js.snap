// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlusIcon renders correctly 1`] = `
<svg
  fill="currentColor"
  id="Plus"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polygon
    points="23 11 23 13 13 13 13 23 11 23 11 13 1 13 1 11 11 11 11 1 13 1 13 11 23 11"
  />
</svg>
`;
